helpers/auth/admin_roles.py

```diff
@@ -19,6 +19,14 @@ def create_rooms_update_delete_office(self, office_id):
         if admin_details.location != location.name:
             raise GraphQLError("You are not authorized to make changes in " + location.name)  # noqa: E501
 
+    def check_location(self, location_id):
+        admin_details = get_user_from_db()
+        location = Location.query.filter_by(id=location_id).first()
```

Comment:
I think for this you filter you should also add state=active to ensure you do not capture locations that have been archived.

Reply:
Thanks, @caveinn, this is being implemented in the different classes. I also had brought it up but @Bonifase emphasised during the sync that we need to prioritise the other tasks and since its implementation is okay. We can always create another story for refactoring code.